frontend/src/metabase/query_builder/actions/object-detail.js

```diff
@@ -107,18 +107,26 @@ export const loadObjectDetailFKReferences = createThunkAction(
       const card = getCard(state);
       const queryResult = getFirstQueryResult(state);
 
-      async function getFKCount(card, queryResult, fk) {
-        const fkQuery = Q_DEPRECATED.createQuery("query");
-
-        fkQuery.database = card.dataset_query.database;
-        fkQuery.query["source-table"] = fk.origin.table_id;
-        fkQuery.query.aggregation = ["count"];
-        fkQuery.query.filter = getFilterForFK(objectId, fk);
+      async function getFKCount(card, fk) {
+        const metadata = getMetadata(getState());
+        const databaseId = new Question(card, metadata).databaseId();
+        const tableId = fk.origin.table_id;
```

Comment:
I think I saw a comment from Alex re using `const tableId = question.tableId();`. IDK the difference, but using `fk.origin.table_id` looks a bit verbose. Or are those tableIds different?

Reply:
They are different.